.github/workflows/uberjar.yml

```diff
@@ -18,6 +18,16 @@ on:
     inputs:
       commit:
         description: 'Optional full-length commit SHA-1 hash'
+  # This workflow will be called from "release-embedding-sdk.yml"
+  workflow_call:
+    inputs:
+      commit:
+        description: 'Optional full-length commit SHA-1 hash'
+        type: string
+      only_ee:
+        description: A flag to indicate if only EE Uberjar should be built.
+        type: boolean
+        default: false
```

Comment:
Is there a situation where `only_ee` would be `false`? I thought we always only need the EE uberjar

Reply:
For this case, no. `workflow_call` is only being called from the SDK. But I think it's great to be explicit, so folks who care about `uberjar.yml` could know that it should behave the same by building 2 jars for all existing use cases.

Also, this corresponds to `workflow_dispatch`'s `inputs.only_ee` which should be evaluated to falsy value since `only_ee` isn't defined in `workflow_dispatch`